minotaur.pro

```diff
@@ -26,15 +27,26 @@ SOURCES += \
     code/gui/mainwindow.cpp \
     code/utility/clock_time.cpp \
     code/utility/logger.cpp \
-    code/gui/actuatorsetup.cpp
+    code/gui/actuatorsetup.cpp \
+    code/controller/simulator.cpp \
+    code/gui/simulatorwindow.cpp \
+    code/utility/vector2i.cpp \
+    code/gui/graphics/simulatorscene.cpp \
+    code/gui/graphics/robotgraphicsitem.cpp

```

Comment:
don't make a graphics folder inside gui. Either separate the GUI -> graphics logic and put the graphics files in `code/graphics` or just put the graphics related files in the gui folder directly, if the logic can't be separated.